src/lang/ru.rs

```diff
@@ -30,15 +30,15 @@ pub static ref T: std::collections::HashMap<&'static str, &'static str> =
         ("IP Whitelisting", "Список разрешенных IP-адресов"),
         ("ID/Relay Server", "ID/Сервер ретрансляции"),
         ("Import Server Config", "Импортировать конфигурацию сервера"),
-        ("Export Server Config", ""),
+        ("Export Server Config", "Экспортировать конфигурацию сервера"),
         ("Import server configuration successfully", "Конфигурация сервера успешно импортирована"),
-        ("Export server configuration successfully", ""),
+        ("Export server configuration successfully", "Конфигурация сервера успешно экспортирована"),
         ("Invalid server configuration", "Недопустимая конфигурация сервера"),
         ("Clipboard is empty", "Буфер обмена пуст"),
         ("Stop service", "Остановить службу"),
         ("Change ID", "Изменить ID"),
         ("Website", "Веб-сайт"),
-        ("About", "О RustDesk"),
+        ("About", "О программе"),
```

Comment:
Just fix of low priority cosmetic bug
![image](https://user-images.githubusercontent.com/1307229/198153826-1f2e0fc8-84c4-45ab-8c53-50b343f491f8.png)

It will be "О программе RustDesk"